### ISSUE TITLE
[Feature] Add share/posting function for Bluesky App

### ISSUE THREAD

**Issue by perry-sim** [FreshRSS/FreshRSS#5706]
Because Musk rides Twitter/X fürther and further into the abyss, it would be great to have an addtional [sharing service](https://freshrss.github.io/FreshRSS/en/users/08_sharing_services.html) for the [Bluesky App](https://bsky.app/). 

See  [Posting via the Bluesky API](https://atproto.com/blog/create-post)

**Comment by math-GH** [comment]
As far as I see the API uses POST parameters instead of GET.

(Here is help needed. I am not familiar with POST)

**Comment by Frenzie** [comment]
I don't think this is compatible with the way the sharing functionality is currently implemented. Not because of the POST; that's just method POST instead of GET, but because it requires constructing a session first.
